scripts/database.js

```diff
@@ -76,6 +76,17 @@ export class Database {
         await store.put(tx, tx.txid);
     }
 
+    /**
+     *
+     * @param {String} keyToExport - A key that uniquely identifies a wallet
+     */
+    async storeTxDbIdentifier(keyToExport) {
+        const store = this.#db
+            .transaction('txs', 'readwrite')
+            .objectStore('txs');
+        await store.put(keyToExport, 'keyToExport');
+    }
```

Comment:
I don't like adding a special object to the object store, when we add the multiple account system, this whole PR would need to be rewritten.
Instead, we can start populating the `account` object store with keyToExport (Or publicKey as it's called there) as soon as we get an import and link the deletion of one to the other.